packages/core/src/Flex/README.mdx

```diff
@@ -0,0 +1,107 @@
+---
+title: 'Flex'
+type: 'component'
+slug: /components/flex/
+github: 'https://github.com/contentful/forma-36/tree/master/packages/forma-36-react-components/src/components/Flex'
+storybook: 'https://f36-storybook.contentful.com/?path=/story/components-flex--default'
+---
+
+Flex components is [Box] with `display: flex` and abilities to apply other flexbox CSS properties via properties of the component.
+
+The Box component is useful because it helps with several common cases:
+
+- Create a layout with uses Forma36 spacing
+- Compose new component and allow to override element using the `as` property.
```

Comment:
Should we mention that it's meant to be used for one direction layouts vertical or horizontal?

Reply:
good call, updated!